beacon-light-client/plonky2/validators_commitment_mapper_tree/utils.ts

```diff
@@ -48,7 +48,7 @@ export function* makeBranchIterator(indices: bigint[], depth: bigint) {
   }
 }
 
-function bitArrayToByteArray(hash: number[]): Uint8Array {
+export function bitArrayToByteArray(hash: number[]): Uint8Array {
```

Comment:
You can use directly `bitsToBytes` from `hex-utils.ts` which returns a hex (maybe also rename it) as you use the function to convert to hex actually